openlibrary/templates/messages.html

```diff
@@ -9,8 +9,8 @@
 $ comment_new_work = _("Created new work record.")
 $ comment_add_book = _("Added new book.")
 
-$ flash_book_added = _("Thank you very much for adding that new book!")
-$ flash_book_updated = _("Thank you very much for improving that record!")
-$ flash_work_updated = _("Thank you very much for improving that record!")
+
+$ flash_catalog_updated = _("Thank you very much for improving that record!")
```

Comment:
```suggestion
$ flash_catalog_updated = _("Thank you for improving the Open Library catalog!")
```
As per the requirements ([1](https://github.com/internetarchive/openlibrary/issues/10674#issuecomment-2802842602), [2](https://github.com/internetarchive/openlibrary/pull/10881#pullrequestreview-2889420606))